bugbug/repository.py

```diff
@@ -456,6 +500,15 @@ def download_commits(repo_dir, date_from):
             for path_prev in prev_commit.files:
                 files_touched_90_days[path_prev] += 1
 
+            directories_prev = set(
+                directory
+                for path in prev_commit.files
+                for directory in os.path.dirname(path).split("/")[:2]
+                if directory != ""
+            )
```

Comment:
This will extract each directory of level 0 and level 1 independently. So a modification on a file `docshell/build/nsDocShellModule.cpp` and a modification on a file `image/build/moz.build` will both haves the same `build` feature. Is it expected? I don't know about the mozilla-central architecture to know if it makes sense or not so please forgive me if my question is silly.

Reply:
No it's not expected, e.g. for `docshell/build/nsDocShellModule.cpp` we should have `docshell` ad `docshell/build`.